vue/components/ui/atoms/image-item/image-item.vue

```diff
@@ -40,9 +40,10 @@
             class="item-image"
             v-bind:style="style"
             v-bind:class="{ 'dropdown-open': optionsVisible }"
+            ref="image"
             v-on:animationend="onAnimationEnd"
         >
-            <image-ripe v-bind:style="imageStyle" v-bind:src="imageUrl" v-bind:alt="name" />
+            <image-ripe v-bind:style="style" v-bind:src="imageUrl" v-bind:alt="name" />

```

Comment:
Keep calling it `imageStyle` as we reserve `style` for when it's just for the root component (the div, in this case)

Reply:
It will result in two computed that have the same code